capstone/src/main/java/com/google/sps/data/ReadData.java

```diff
@@ -61,23 +111,28 @@ public void readCSV() {
         }
 
         if (inStore == null) {
+          entity.setProperty("zipCode", zip);
           entity.setProperty("id", id);
           entity.setProperty("date", date);
           entity.setProperty("completion", completion);
           entity.setProperty("gender", gender);
           entity.setProperty("ageRange", ageRange);
           entity.setProperty("answerOne", answerOne);
           entity.setProperty("answerTwo", answerTwo);
```

Comment:
Maybe make the property names more descriptive (so rating, exposure, and text instead of answerOne answerTwo answerThree)?